examples/tracer-web/examples/xml-http-request/index.js

```diff
@@ -0,0 +1,60 @@
+import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { WebTracer } from '@opentelemetry/web';
+import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
+import { ZoneScopeManager } from '@opentelemetry/scope-zone';
+import { CollectorExporter } from '@opentelemetry/exporter-collector';
+
+const webTracerWithZone = new WebTracer({
+  scopeManager: new ZoneScopeManager(),
+  plugins: [
+    new XMLHttpRequestPlugin({
+      ignoreUrls: ['http://localhost:8090/sockjs-node/info'],
+      propagateTraceHeaderUrls: [
+        'http://localhost:8090'
```

Comment:
I think we should probably propagate trace headers to same-origin URLs by default. We did that in OpenCensus Web, see https://github.com/census-instrumentation/opencensus-web/blob/68aa988732e03f2de49ebc0213c3b2525aa5b68b/packages/opencensus-web-instrumentation-zone-peer-dep/src/util.ts#L26

The reason for my suggestion is that same-origin requests can attach any headers without any performance penalty, because it's generally assumed from a browser security model that server sending the site's HTML is trusted.